02-starting-with-data.Rmd

```diff
@@ -290,19 +288,14 @@ feature to get the full and correct column name.
 ```
 
 When we did `str(surveys)` we saw that several of the columns consist of
-integers, however, the columns `genus`, `species`, `sex`, `plot_type`, ... are
-of a special class called a `factor`. Factors are very useful and are actually
-something that make R particularly well suited to working with data, so we're
+integers. The columns `genus`, `species`, `sex`, `plot_type`, ... however, are
+of a special class called `factor`. Factors are very useful and actually
+contribute to making R particularly well suited to working with data. So we are
 going to spend a little time introducing them.
 
-Factors are used to represent categorical data. Factors can be ordered or
-unordered, and understanding them is necessary for statistical analysis and for
-plotting.
-
-Factors are stored as integers, and have labels (text) associated with these
-unique integers. While factors look (and often behave) like character vectors,
-they are actually integers under the hood, and you need to be careful when
-treating them like strings.
+Factors represent categorical data. They are stored as integers associated with labels. While factors look (and often behave) like character vectors,
```

Comment:
Why do you suggest removing the part about ordered and unordered factors?

Reply:
I was trying to declutter this section a little. To keep the idea of odered and unordered, how about "Factors represent categorical data. They are stored as integers associated with labels and they can be ordered or unordered . While factors look (and often behave) like character vectors,